packages/ActionBar/src/hooks/useSort/index.js

```diff
@@ -1 +1 @@
-export { default } from "./useSort";
+export { default, sortData } from "./useSort";
```

Comment:
no biggie but we should figure out how to properly export this kind of helper functions in all our ecosystems eventually.

To me we shouldn't expose it here, we should do `import {sortData} from @paprika/action-bar/lib/useSort` 

but 🤷  :D